src/help.ts

```diff
@@ -52,4 +55,58 @@ export abstract class BBHelpClient {
   public static toggleOpen(): void {
     BBHELP.HelpWidget.toggleOpen();
   }
+
+  public static openWidget(): void {
+    BBHELP.HelpWidget.open();
+  }
+
+  public static closeWidget(): void {
+    BBHELP.HelpWidget.close();
+  }
+
+  public static disableWidget(): Promise<any> {
+    return BBHelpClient.ready()
```

Comment:
Nothing too major, but now you're intruding both async and sync methods without a clear naming difference between them (e.g., `fs` has both `readFile` and `readFileSync`). Curious what your thoughts are on `disableWidgetAsync` or something similar? (Or, we could just make all the methods sync to make the documentation easier...) @blackbaud-brandonhare You have an opinion?

Reply:
The only thing I don't like about the separate names is having a method `disableWidgetAsync` would kind of imply there is a `disableWidget` Sync method.  I think of the options, I would prefer making all the methods sync with a note about the ready check.

Though I also think there would be benefit in making async versions of all methods...